pkgs/development/go-modules/generic/default.nix

```diff
@@ -20,6 +20,9 @@
 , vendorSha256 ? null
 # Whether to delete the vendor folder supplied with the source.
 , deleteVendor ? false
+# Whether to run the vend tool to regenerate the vendor directory.
+# This is useful if any dependency contain C files.
+, runVend ? false
```

Comment:
what do you think of using `useVend`?

Reply:
We had this before https://github.com/NixOS/nixpkgs/pull/89453#discussion_r435156648 I thought about something more descriptive without having a better idea.

Reply:
Sounds like this is staying runVend without a 3rd alternative.